src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,59 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (this.constructor.name !== 'Herbivore') {
+      Animal.alive.push(this);
+    }
```

Comment:
Why avoid `push`ing Herbivores to alive in the `Animal` `constructor` and have it re-implemented in the `Herbivore` `constructor`?

Avoid implementing the same logic in multiple places!

Reply:
cause i need to add Herbivore hidden key